MemeifyMe/app/src/main/res/layout/activity_main.xml

```diff
@@ -9,9 +9,52 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context=".MainActivity">
 
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="40dp"
+        android:layout_marginTop="20dp"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="Memifier"
```

Comment:
Let's minimize hard-coded strings by keeping these resources in our strings.xml, yeah?